react/box-bitcoin.jsx

```diff
@@ -49,7 +49,16 @@ let BoxBitcoin = React.createClass({
 
   monitorDattCore: function () {
     let dattcore = this.props.dattcore
+    let initBalances = dattcore.getLastBalances()
+    if (initBalances) {
+      this.handleBitcoinBalance(initBalances)
```

Comment:
The idea of the "monitor" function is to just have all the .on methods. There should probably be a different function for "emit initial events". Then the handle functions will handle them like normal. But then again, I'm not sure the UI should be emitting any initial events. The initial data should just be plugged into the props, no?